crl-test/service_accounts.tf

```
@@ -1,8 +1,14 @@
-# Service account for the CRL Integration tests to use.
-resource "google_service_account" "crl" {
+# Service accounts for the CRL Integration tests to use.
+resource "google_service_account" "crl-admin" {
```

Please make all Terraform resource names use underscores instead of dashes.
```suggestion
resource "google_service_account" "crl_admin" {
```